backend/utils.py

```diff
@@ -16,3 +16,16 @@ def wrapped_view(**values):
         return jsonify(view(**values))
 
     return wrapped_view
+
+# formatter that adds information about url and ip to logs
+class RequestFormatter(logging.Formatter):
```

Comment:
You are simply using the `Formatter` only, then why not import that particular Object instead of importing the whole `logger` ?